Dikke Ploaten/Services/Database.swift

```diff
@@ -204,11 +204,11 @@ class Database {
 	}
 	
 	// Upload image
-	func uploadImage(image: UIImage, isImgProfile: Bool) {
+	func uploadImage(image: UIImage, isPickingImageForProfile: Bool) {
```

Comment:
Hier was het wel goed, hier ga je zeggen "ik wil een image uploaden voor de user", en daar heb je geen informatie aan dat deze gepicked is. Maak er misschien om volledig juist te zijn `asProfileImage` of zo van.